src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -48,8 +49,10 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Photo photo = ParserUtil.parsePhoto(argMultimap.getValue(PREFIX_PHOTO).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
-        Remark remark = new Remark(); // add command does not allow adding remarks straight away
```

Comment:
Same as below, I think Jiatong recently added this so I think we should leave it. Would it be possible to add that back in?

Reply:
Done too